sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -667,6 +668,42 @@ public static void endSession() {
     return getCurrentHub().getSpan();
   }
 
+  /**
+   * Starts a Transaction being a continuation of the span described in a `header` parameter.
+   * Started transaction is set on the scope.
+   *
+   * @param name the transaction name
+   * @param operation the operation
+   * @param header the string representing {@link SentryTraceHeader}
+   * @return created transaction
+   */
+  public static ISpan continueFromHeader(
+      final @NotNull String name, final @NotNull String operation, final @NotNull String header)
+      throws InvalidSentryTraceHeaderException {
```

Comment:
not sure if I like the idea of `InvalidSentryTraceHeaderException`, so every call to this method would need to be wrapped within a try catch, the other option would be to swallow it and return a NoOpTransaction

wdyt @bruno-garcia ?

Reply:
Or start a new transaction without a parent context

Reply:
Agree it isn't great to throw. 
There's not much the user can do when they catch that other than "log with Sentry?"

Returning a new trace without a parent is likely better but best would be to log an error so when the time comes to debug this, we can logs. Log could print out the broken header too.

Reply:
wfm, not sure if useful but the new transaction could have a tag or extra or whatsoever with the invalid header so they know what happened if its not linked at all